src/recensio/plone/behaviors/editorial.py

```diff
@@ -66,3 +69,22 @@ def editorial(self):
     @editorial.setter
     def editorial(self, value):
         self.context.editorial = value
+
+    def get_formatted_editorial(self):
+        editors_list = [
+            getFormatter(" ")(editor.to_object.firstname, editor.to_object.lastname)
+            for editor in self.context.editorial
+        ]
```

Comment:
This looks safer,  does not call twice `to_object` for every iteration and does initialize `getFormatter(" ")` only once:
```suggestion
        formatter = getFormatter(" ")
        editors_list = []
        for item in self.context.editorial:
            obj = item.to_object
            if obj:
                editors_list.append(formatter(obj.firstname, obj.lastname))
```